modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -91,17 +91,17 @@ $ oc adm upgrade
 [source,terminal]
 ----
 NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
-version   4.11.18   True       False         6m29s   Cluster version is 4.11.18
+version   4.<y+1.z>   True       False         6m29s   Cluster version is 4.<y+1.z>
 ----
 
-. Update to version 4.12 by running the following command:
+. Update to version 4.y+2 by running the following command:
```

Comment:
Hmm, you made 4.<y+1.z> where y+1.z is in the brackets, while 4.y+2 doesn't have brackets. We need to make them consistent.

Reply:
good call :)